var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -219,7 +219,6 @@ class Dealii(CMakePackage, CudaPackage):
           when='@9.0.0:9.1.1')
 
     # Check for sufficiently modern versions
-    conflicts('cxxstd=98', when='@9.0:')
```

Comment:
Sorry that I didn't see your review request until now @alalazo. Thanks a lot for the fix!